reference/method-patterns.md

```diff
@@ -4,54 +4,87 @@ description: A simple and powerful way of identifying method definitions and inv
 
 # Method patterns
 
-Many recipes operate on method declarations or invocations. Method patterns are a mechanism for configuring and developing such recipes. OpenRewrite's pointcut method expressions are derived from AspectJ. An OpenRewrite "method pattern" is comparable to an AspectJ's "Pointcut method expression".
+Traditionally, searching for or identifying methods was done by doing a simple text search for the name of the method or the parameters. You'd then look through all the results and ponder if this is _actually_ the method you wanted to find or not. When searching over a significant number of repositories, this becomes an impossible task, though. There could be hundreds of methods with the same name and it would be too much work to find or change just the ones you cared about.
+
+Fortunately, OpenRewrite offers a better alternative: method patterns. Method patterns provide a way to accurately and quickly identify one or more method definitions or invocations. These patterns can be used during the creation of recipes or by users when they run recipes.
```

Comment:
I'd maybe swap these two paragraphs around, as we sometimes link to the page, and then people want to quickly skim the first paragraph to see what this is about.

Reply:
Ended up re-writing the intro to be shorter. Hopefully that addresses your concern.

Reply:
Thanks a lot!